lib/tasks/copyright.rake

```diff
@@ -101,6 +103,15 @@ namespace :copyright do
     end
   end
 
+  def missing_copyright_regexp(format)
+    case format
+    when :ruby, :rb
+      /\A(?<shebang>#![^\n]+\n\n?)?(?<additional># frozen_string_literal: (?:true|false)\n\n?)?\n*/m
+    else
+      raise "Format #{format} is not yet supported for copyright creation"
```

Comment:
I suggest not failing, maybe returning `nil` and warning either here or where it is used, to be able to run it on whole codebase

Reply:
I just copied the pattern that we use for all other case statements as well. It just makes sure that you (the rake programmer) don't call `rewrite_copyright` with `create: true` for a format that doesn't support create. The same way that the other case statements make sure that you don't call it with an entirely unsupported format.

Or am I overlooking an existing rake task that now doesn't work anymore due to this change?

Reply:
Please ignore, I misunderstood the logic